website/docs/devtools/pull-requests.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Hi @PaulD-docs let's remove this file and all associated images, we don't need it in the docs portal